.github/workflows/ci.yml

```
@@ -11,57 +11,64 @@ permissions:
           load: true
-          tags: dotfiles-test:ci
+          tags: dotfiles-test:${{ matrix.profile }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
           build-args: |
             CHEZMOI_BRANCH=${{ github.head_ref || github.ref_name }}
+            CHEZMOI_PROJECT=${{ matrix.profile == 'core' && '' || matrix.profile }}
 
-      - name: Verify core tools are installed
+      - name: Verify profile via scripts/test-profile.sh
         run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)